### PR TITLE
remove redundancy codes in zepto.init

### PR DESCRIPTION
**Pull request by codeudan**
In function which called zepto.init,some codes is redundancy.That codes never be executed,the codes of create DOM fragment has written above codes that I deleted.Thanks!

### CODE REVIEW

File: src/zepto.js [hunk 1]
```diff
@@ -212,9 +212,6 @@ var Zepto = (function() {
       // Wrap DOM nodes.
       else if (isObject(selector))
         dom = [selector], selector = null
-      // If it's a html fragment, create nodes from it
-      else if (fragmentRE.test(selector))
```
Comment (mislav) [inline src/zepto.js]: Good catch! 👍

File: src/zepto.js [hunk 2]
```diff
@@ -716,7 +713,7 @@ var Zepto = (function() {
         var element = this[0]
         if (typeof property == 'string') {
           if (!element) return
-          return element.style[camelize(property)] || getComputedStyle(element, '').getPropertyValue(property)
+          return element.style[camelize(property)] || getComputedStyle(element, '').getPropertyValue(dasherize(property))
```
Comment (mislav) [inline src/zepto.js]: This is a feature change. Ideally, a code cleanup and an unrelated feature change should be submitted in separate PRs. However, for now it's fine. Please add a test with what this accomplishes. Thanks!

Comment (mislav) [inline src/zepto.js]: I just noticed that there is another place where we call `computedStyle.getPropertyValue(prop)` a few lines below, and that could benefit from the same fix that you did here. You would also need to add a new test for the `.css(["fooBar", "bazKuux"])` invocation style. Thanks!

File: src/zepto.js (hunk 1 above)
Comment (peacelee) [inline src/zepto.js]: why the "else if (fragmentRE.test(selector))" did not remove?
